.github/actions/assign-on-ptal/main.py

```diff
@@ -0,0 +1,122 @@
+import os
+import requests
+import json
+import re
+
+def findAssignees (comment):
+    """
+    This function is used to extract the usernames present in the form 
+    `@person1 @person2 ptal!` anywhere in a comment
+
+    Args: 
+        comment - Input multiline string
```

Comment:
This isn't the correct format for defining Args. Pleaase follow our python [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python) and refer other python files.